package.json

```diff
@@ -23,7 +23,7 @@
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
     "@mdx-js/react": "^1.6.22",
-    "@mui/material": "^5.10.3",
```

Comment:
Updating this module does not fix any item address at https://github.com/karmada-io/website/issues/79.
Why are we going to do this?

Reply:
this is a small dependency update
if not needed, then I will close this PR

Reply:
Since there are few descriptions, I can't tell why we need this update.